app/collections/manager.cjsx

```diff
@@ -17,7 +17,12 @@ module?.exports = React.createClass
     subject: React.PropTypes.object
     project: React.PropTypes.object
 
+  logModalChange: (clickedItem) ->
+    @props.contextRef?.geordi?.logEvent
```

Comment:
Probably because this is in a modal. Modals don't have access to context. There are a couple instances of this in the Geordi logging.